Description: Upstream changes introduced in version 0.2-3
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 covetel-peribeco (0.2-3) unstable; urgency=low
 .
   * variables de entorno de la aplicacion
 .
 The person named in the Author field signed this changelog entry.
Author: Luis Alfredo da Silva <luisdasilva@covetel.com.ve>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- covetel-peribeco-0.2.orig/Peribeco/script/peribeco.psgi
+++ covetel-peribeco-0.2/Peribeco/script/peribeco.psgi
@@ -2,13 +2,12 @@
 use strict;
 
 use Plack::Builder;
-use PERIBECO;
+use Peribeco;
 
-PERIBECO->setup_engine('PSGI');
-my $app = sub { PERIBECO->run(@_) };
+my $app = Peribeco->psgi_app( @_ );
 
 builder {
     enable_if { $_[0]->{REMOTE_ADDR} eq '127.0.0.1' }
         "Plack::Middleware::ReverseProxy";
     $app;
-};
+}
--- covetel-peribeco-0.2.orig/Peribeco/lib/Peribeco.pm
+++ covetel-peribeco-0.2/Peribeco/lib/Peribeco.pm
@@ -31,7 +31,7 @@ use Catalyst qw/
 
 extends 'Catalyst';
 
-our $VERSION = '0.01';
+our $VERSION = '0.2-3';
 $VERSION = eval $VERSION;
 
 # Configure the application.
@@ -49,9 +49,9 @@ __PACKAGE__->config(
     disable_component_resolution_regex_fallback => 1,
 );
 
-__PACKAGE__->config(
-    'Plugin::ConfigLoader' => { file => 'configuracion.yml' },
-);
+#__PACKAGE__->config(
+#    'Plugin::ConfigLoader' => { file => 'configuracion.yml' },
+#);
 
 # Start the application
 __PACKAGE__->setup();
